index.html

```diff
@@ -0,0 +1,158 @@
+                    <path d="M-11.8095 0.641908C-13.8144 5.33769 -13.8718 10.6376 -11.969 15.3757C-10.0663 20.1137 -6.35924 23.9019 -1.66346 25.9068C3.03231 27.9117 8.3322 27.969 13.0703 26.0663C17.8084 24.1635 21.5965 20.4565 23.6014 15.7607L-11.8095 0.641908Z" fill="#6070FF"/>
+                </svg>                
+            </div>
+            <div class="emptycircle">
+                <svg width="15" height="15" viewBox="0 0 15 15" fill="none" xmlns="http://www.w3.org/2000/svg">
+                    <circle cx="7.5" cy="7.5" r="7" fill="white" stroke="#CBCDFF"/>
+                 </svg>              
+            </div>
+            <div class="circle">
+                <svg width="11" height="12" viewBox="0 0 11 12" fill="none" xmlns="http://www.w3.org/2000/svg">
+                    <ellipse cx="5.30398" cy="6.21088" rx="5.5" ry="5" transform="rotate(52.9708 5.30398 6.21088)" fill="#CBCDFF"/>
+                </svg>                
+            </div>
+            <div class="siderec">
+                <svg width="9" height="44" viewBox="0 0 9 44" fill="none" xmlns="http://www.w3.org/2000/svg">
+                    <rect x="-34" y="37.4868" width="51.9898" height="9.30073" transform="rotate(-46.1415 -34 37.4868)" fill="#EBEBFF"/>
+                 </svg>      
+            </div>
+        
+    </section>
+
+    <!-- the text part -->
+
+    <section class="text">
+        <h1>I’m Mariane
+            Glad to see you!
+        </h1>
+        <p>I’m a software developer! I can help you 
+            build a product , feature or website Look
+             through some of my work and experience! 
+             If you like what you see and have a project 
+             you need coded, don’t hestiate to contact me.
+        </p>
+        <div class="social-text">
+            <p><b>LET'S CONNECT</b></p>
+        </div>
+        <div class="social-media">
+            <ul>
+                <li>
+                    <div class="twitter">
+                        <svg width="20" height="20" viewBox="0 0 20 20" fill="none" xmlns="http://www.w3.org/2000/svg">
+                            <path d="M19.7587 0.394423C19.4253 -0.00424693 18.8587 -0.103914 18.4253 0.195088L16.492 1.52399C15.592 0.560536 14.3253 0.0289756 12.9587 0.0289756C10.2587 0.0289756 8.02532 2.22166 8.02532 4.94591C8.02532 5.34458 8.05865 5.71002 8.15865 6.07547C5.82532 5.57713 4.42532 3.88279 3.89198 0.826316C3.82532 0.427646 3.52532 0.128643 3.12532 0.0289756C2.75865 -0.070692 2.35865 0.0954206 2.12532 0.394423C2.05865 0.460868 -3.44135 8.10205 4.52532 14.5804C3.75865 16.2083 1.45865 17.6037 0.558651 18.0688C0.125318 18.2681 -0.0746825 18.7332 0.0253175 19.1984C0.125318 19.6635 0.525318 19.9625 0.991984 19.9625H2.72532C12.8587 19.9625 17.4587 14.9459 17.5587 3.71667L19.6587 1.72332C20.0587 1.3911 20.092 0.793093 19.7587 0.394423Z" fill="#505F79"/>
+                        </svg>                 
+                    </div>
```

Comment:
- [ ] I can see that you group your icons by using `ul`>`li` tags but as required for this project each icon should use an `a` tag, the `a` tag will let you connect your social networks with the icons, using href , kindly fix this :wink:  :wink: